setup.sh

```diff
@@ -521,21 +537,22 @@ cat << EOF
 
 Open Grafana at http://localhost:9000/ ... use admin/admin for login.
 
-Follow these instructions for *Grafana Setup*:
+To complete *Grafana Setup*:
 
-* From 'Configuration\Data Sources' add 'InfluxDB' database with:
-  - Name: 'InfluxDB'
+* From 'Dashboard\Browse', select 'New/Import', browse to ${PWD}/dashboards
+  and upload ${DASHBOARD}.
+
+NOTE: The datasources for InfluxDB and SunAndMoon are already configured.
+If you need to modify them via Configuration\Data Sources:
+
+* InfluxDB
   - URL: 'http://influxdb:8086'
   - Database: 'powerwall'
   - Min time interval: '5s'
   - Click "Save & test" button
 
-* From 'Configuration\Data Sources' add 'Sun and Moon' database with:
-  - Name: 'Sun and Moon'
+* Sun and Moon
   - Enter your latitude and longitude (tool here: https://bit.ly/3wYNaI1 )
   - Click "Save & test" button
 
-* From 'Dashboard\Browse', select 'New/Import', browse to ${PWD}/dashboards
-  and upload ${DASHBOARD}.
-
```

Comment:
Updated the setup instructions.

Reply:
That's good, although the problem with provisioned data sources is that they cannot be edited within the UI anymore.  They would need to edit the YML.

![image](https://github.com/user-attachments/assets/3181a700-8ec5-4e5f-980d-3a2cb59a0ba7)

![image](https://github.com/user-attachments/assets/3c563875-2503-4a99-9ea5-dce66b4a638f)

(Although technically we can enable `editable: true` but that will get overwritten the next time the docker container is restarted)

Reply:
Ah, that's a good point.  In fact, that makes me re-think how we approach this. As it is now, anyone who does an upgrade will get the InfluxDB provisioning. That could break installations that were customized.

I have a thought... Let's move the influxdb setup to a template file like the sunandmoon.yml file and have it install as part of the setup run. In fact, I suggest we add code to detect those yml files and if they are already installed, notify or ask user if they want them overwritten.

I believe that would allow for customization (even if not via the UI) that is not overwritten by upgrades. We can adjust the setup instruction to point the user to editing those files in the /grafana/provisions/datasources folder.

Reply:
Good point. Based on my testing Grafana seems to overwrite the data source based on name (even if I give it a custom `uid`).

I think we can rename both the InfluxDB and Sun and Moon auto provisioned data source to "InfluxDB (auto provisioned)" and "Sun and Moon (auto provisioned)".

This means it **won't** overwrite any existing installs with customizations, and also users can easily identify which ones are their own data sources and which ones are the automatic ones.